WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -113,14 +139,20 @@ struct NewOrder: View {
         .navigationTitle(Localization.title)
         .navigationBarTitleDisplayMode(.inline)
         .toolbar {
+            ToolbarItem(placement: .cancellationAction) {
```

Comment:
This item should be added only when the `.splitViewInOrdersTab` feature flag is enabled.